plugins/wme-sitebuilder/wme-sitebuilder/Support/ConsoleCommand.php

```diff
@@ -176,21 +178,14 @@ protected function getWpBinary() {
 			return self::$wpBinary;
 		}
 
-		/*
-		 * Construct an escaped string that expands the current PHP and WP-CLI binary paths.
-		 *
-		 * Note that we're using the PHP_BINDIR constant and adding "/php" instead of PHP_BINARY,
-		 * as the latter will point to PHP-FPM.
-		 *
-		 * The expected output of this will look something like:
-		 *
-		 *     /opt/remi/php73/root/usr/bin/php /usr/local/bin/wp
-		 */
+		// Set the wp-cli cache directory to a known writable location.
+		putenv( sprintf( 'WP_CLI_CACHE_DIR=%s.wp-cli/cache', get_temp_dir() ) );
+
+		// Run wp cli from our vendor folder, this is actually a bash script that calls wp under the hood.
 		self::$wpBinary = sprintf(
 			'%1$s %2$s',
-			escapeshellarg( PHP_BINDIR . '/php' ),
-			// phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.system_calls_shell_exec
-			escapeshellarg( trim( (string) shell_exec( 'command -v wp' ) ) )
+			escapeshellarg( VENDOR_DIR . 'wp-cli/wp-cli/bin/wp' ),
```

Comment:
I'm 99% sure this isn't an issue, but we'll need to make sure that using this version of WP CLI will not conflict or cause any problems with the version installed on Nexcess. Prior to this, we would have been tying into the version of WP CLI provided by the host, so wee need to confirm that won't be an issue if we modify the behavior.

Reply:
As mentioned on slack, there is only one other place that is using this for the moment in [Shippping.php](https://github.com/moderntribe/wme/blob/e4036a642c567be4255c10bd8f15eecd218d2d7f/plugins/wme-sitebuilder/wme-sitebuilder/Wizards/Shipping.php#L143) so I suggest we just convert that to use the PHP installer since it's runs much faster and works locally on all operating systems in a separate PR after this merged.